mne/filter.py

```diff
@@ -2018,6 +2018,9 @@ def resample(self, sfreq, npad='auto', window='boxcar', n_jobs=1,
                  pad='edge', verbose=None):  # lgtm
         """Resample data.
 
+        If appropriate, an anti-aliasing filter is applied before resampling.
+        See :ref:`resampling-and-decimating` for more information.
```

Comment:
What do you think about this one @cbrnr?

```suggestion
        Through use of an FFT-based (fast Fourier transform) method for
        resampling, aliasing artifacts are prevented as if an anti-aliasing
        filter were applied beforehand.
        See :ref:`resampling-and-decimating` for more information.
```